os/network-config-with-networkd.md

```diff
@@ -86,6 +86,18 @@ DHCP=yes
 
 To apply the configuration, run `sudo systemctl restart systemd-networkd`. Check the status with `systemctl status systemd-networkd` and read the full log with `journalctl -u systemd-networkd`.
 
+## Debug networkd
+
+If you've faced some problems with networkd you can enable debug mode following the instructions below.
+
+```sh
+mkdir -p /etc/systemd/system/systemd-networkd.service.d/
+echo -e "[Service]\nEnvironment=SYSTEMD_LOG_LEVEL=debug" > /etc/systemd/system/systemd-networkd.service.d/10-debug.conf 
```

Comment:
Can we expand this out to show it as

```
[Service]
Environment=SYSTEMD_LOG_LEVEL=debug
```

Doing all of that inline is a little harder to understand.